include/z64actor.h

```diff
@@ -1137,8 +1144,8 @@ typedef enum {
 #define ACTOR_FLAG_20            (1 << 5)
 // 
 #define ACTOR_FLAG_40            (1 << 6)
-// Invisible
-#define ACTOR_FLAG_80            (1 << 7)
+// hidden or revealed by Lens of Truth (depending on room lensMode)
+#define ACTOR_FLAG_REACT_TO_LENS (1 << 7)
```

Comment:
worth noting that this name has some slight pushback in the ages old oot PR https://github.com/zeldaret/oot/pull/1122#discussion_r858936878

Though I still think its a fine name.

Reply:
:+1: this name

Reply:
Possibly `AFFECTED_BY_LENS` is better?

Reply:
I'm not sure how that really changes anything except being a longer name.

Using the lens is an action "itemAction" and actors with this flag will have a "reaction" to this lens "action"?

Reply:
I think the point is "react" sounds like something the actor physically does (like if it ran away or something when you saw it with Lens), whereas "affected" sounds more like something that occurs more passively in the system itself.

Reply:
Actors (their code) does have to react to lens though, namely drawing to xlu instead of opa (at least in oot) when lens is equipped
Sure you can argue "react" should be gameplay instead maybe but I think it's not super objective of an argument

Reply:
That makes it sound like this doesn't do anything at all, then.

Reply:
Ok to be clear, the way I see it:

Example: fake wall actor
Actor tells z_actor it is lens-aware with the actor flag `ACTOR_FLAG_REACT_TO_LENS`
when time comes to draw it, if `LENS_MODE_HIDE_ACTORS` and lens is active then the actor isn't drawn but instead pushed on `invisibleActors`
`invisibleActors` are drawn later, when the actor's Draw is actually called it draws to xlu if lens is active, opa otherwise

Both sides are aware of the actor being lens-aware, and they cooperate
and since the behavior depends primarily on lens being active, well I'd say it reacts to lens

Reply:
`ACTOR_FLAG_LENS_AWARE` :D 

Personally I prefer `REACT_TO_LENS` over `AFFECTED_BY_LENS` too, but the latter sounds good too, so either way is fine to me (just decide one plz)